src/pages/en/services/opnsense.md

```diff
@@ -4,16 +4,16 @@ description: OPNSense Widget Configuration
 layout: ../../../layouts/MainLayout.astro
 ---
 
-API key / secret can be generated via the admin, see [the opnsense docs](https://docs.opnsense.org/development/how-tos/api.html#creating-keys).
+The API key & secret can be generated via the webui, by creating a new user at `System/Access/Users` called `homepage-api`, ensuring `Generate a scrambled password to prevent local database logins for this user.` is checked, and then edit the effective privileges selecting `Diagnostics: System Activity` & `Status: Traffic Graph` only.  Finally, create a new API key which will download an `apikey.txt` file with your key and secret in it.  Paste the contents following the = symbol in both lines to the username and password fields below.
```

Comment:
```suggestion
The API key & secret can be generated via the webui by creating a new user at _System/Access/Users_. Ensure "Generate a scrambled password to prevent local database logins for this user" is checked and then edit the effective privileges selecting **only**:

-  Diagnostics: System Activity
-  Status: Traffic Graph

Finally, create a new API key which will download an `apikey.txt` file with your key and secret in it. Use the values as the username and password fields, respectively, in your homepage config.
```

Reply:
Thanks, yeah, yours definitely looks more readable than mine!  I have no eye for these things at all.